### PR TITLE
Added Russian translation

### PR DESCRIPTION
**Pull request by userXinos**
My humble translation into Russian. 

I don't know how you can check the modal window with the "`additionalInfo`" button, perhaps "Дополнительная информация" takes up a lot of space, then you can then replace it with "Доп. инфа" for compactness

for native speakers:
"Custom CSS" пока не стал трогать, но думаю можно перевести как "Свой/Ваш CSS" ?
"Emotes" вообще нету идей как переводить, думаю пусть так и будет ? перевод как "Эмоции" мне кажется не очень

### CODE REVIEW

File: assets/locales/ru.json [hunk 1]
```diff
@@ -0,0 +1,304 @@
...
+        "maybeLater": "Может быть позже",
+        "updateAvailable": "Обновление доступно",
+        "updateInfo": "Доступно обновление для BetterDiscord's Injector ({{version}}).\n\nВы можете обновить и перезапустить сейчас или позже.",
+        "updateFailed": "Не удалось обновить",
+        "manualUpdate": "Не удается обновить автоматически, загрузите установщик и переустановите его в обычном режиме.\n\n[Download Installer](https://github.com/rauenzi/BetterDiscordApp/releases/latest)",
+        "jqueryFailed": "Не удалось загрузить jQuery",
+        "jqueryFailedDetails": "Не удалось загрузить jQuery, некоторые плагины могут работать некорректно. Действуйте на свой страх и риск."
+    },
+    "WindowPrefs": {
+        "enabledInfo": "Для правильной работы этой опции требуется прозрачная тема. On Windows this may break your aero snapping and maximizing.\n\nЧтобы изменения вступили в силу, необходимо перезапустить Discord. Вы хотите перезапустить сейчас?",
```
Comment (userXinos) [inline assets/locales/ru.json:301]: Не до конца уверен как правильно перевести

File: assets/locales/ru.json [hunk 2]
```diff
@@ -0,0 +1,304 @@
...
+                "autoScroll": {
+                    "name": "Прокрутить до настроек",
+                    "note": "Автоматическая прокрутка к настройкам плагина при нажатии кнопки (только если кнопка находится вне поля зрения)"
+                },
+                "autoReload": {
+                    "name": "Автоматическая загрузка",
+                    "note": "Автоматически загружает, перезагружает и выгружает плагины и темы"
+                },
+                "editAction": {
+                    "name": "Edit Action",
```
Comment (userXinos) [inline assets/locales/ru.json:75]: Не совсем понял о каком редакторе речь

**Review by userXinos — COMMENTED** [review]
Some questions for native speakers to ensure better translation

**Comment by rauenzi** [comment]
Hi I appreciate you going out of your way to add russian translations, I do have to warn you though that the strings are still rather volatile while development is active. We hope to use a crowd-translation tool to make this easier for both sides.

**Comment by lunabunn** [comment]
@rauenzi I was working on a PR for another language but just saw your comment on this. Does this mean I should just drop the PR and wait for further developments on the crown-translation front?

**Comment by rauenzi** [comment]
@lunabunn @userXinos You can add your translations through POEditor by joining here https://poeditor.com/join/project?hash=nRljcnV0ET this is the recommended way now. You will also be able to check periodically in case there are new strings added as BD adds new features.

**Pull request closed without merging**